docusaurus.config.js

```diff
@@ -95,7 +95,7 @@ const config = {
         {
             navbar: {
                 logo: {
-                    alt: 'Atala PRISM logo',
+                    alt: ' Identus logo',
```

Comment:
@essbante-io , @petevielhaber , how about the logo?

Reply:
I will share PNG, SVG